### PR TITLE
Clean up, add some feature, add warning barrier

### PR DESCRIPTION
**Pull request by gorouflex**
@MrLoLf I'm added some warning to prevent some users enter their email, pass or Token into 2 workflows file, add a feature that will check if the user repo is up-to-date or not with the original repo if it run on GitHub Actions ( idk if it is useful ) replace `IsGit` to `GITHUB_ACTIONS` that already set default when run GitHub Actions

### CODE REVIEW

File: requirements.txt
Comment (MrLoLf) [inline requirements.txt:1]: Could you please restore this file, as it's needed for the local installation with python.

**Review by MrLoLf — CHANGES_REQUESTED** [review]
If you restore the requirements.txt, I'll can merge it. It looks good too me. But those are great additions, thanks a lot.

File: requirements.txt
Comment (gorouflex) [reply requirements.txt:1]: done

**Review by MrLoLf — APPROVED** [review]
Looks Good Thanks

**Pull request merged**